setup.py

```diff
@@ -12,7 +12,7 @@
                       'six',
                       'basictracer>=2.2,<2.3',
                       'googleapis-common-protos==1.5.3',
-                      'requests==2.19.1'],
+                      'requests>=2.19,<3.0'],
```

Comment:
`<3.0` is just a guard against major version changes, right?

Reply:
yeah, i figured that's safer